src/sidebar/panel.js

```diff
@@ -27,13 +27,25 @@ let syncingInProcess = false;
 enableSync.setAttribute('title', browser.i18n.getMessage('syncNotes'));
 giveFeedbackButton.setAttribute('title', browser.i18n.getMessage('feedback'));
 giveFeedbackMenuItem.text = browser.i18n.getMessage('feedback');
-giveFeedbackButton.setAttribute('href', SURVEY_PATH);
-giveFeedbackMenuItem.setAttribute('href', SURVEY_PATH);
 
 giveFeedbackButton.addEventListener('dragstart', (e) => {
   e.preventDefault();
 });
 
+giveFeedbackButton.addEventListener('click', (e) => {
+  e.preventDefault();
```

Comment:
You are repeating the same function code for both of these. You should create a new function and use it for both `addEventListener`s